<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="PN-HYDTECH-041" name="com.payswiff_Regression.test" tests="1" failures="0" timestamp="26 Mar 2018 06:27:57 GMT" time="0.099" errors="1">
  <testcase name="login" time="0.099" classname="com.payswiff_Regression.test">
    <error type="java.lang.Exception" message="java.io.IOException: Destination &#039;/home/itadmin/Desktop/sai/mystuff/PayswiffAutomation/testResults/Chrome/26-03-2018/11-57-57/screenshots&#039; exists but is a directory">
      <![CDATA[java.lang.Exception: java.io.IOException: Destination '/home/itadmin/Desktop/sai/mystuff/PayswiffAutomation/testResults/Chrome/26-03-2018/11-57-57/screenshots' exists but is a directory
	at com.payswiff.util.DriverUtilsImpl.takescreenshot(DriverUtilsImpl.java:1551)
	at com.payswiff_Regression.test.login(test.java:42)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1215)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
	at org.testng.TestNG.run(TestNG.java:1048)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
Caused by: java.io.IOException: Destination '/home/itadmin/Desktop/sai/mystuff/PayswiffAutomation/testResults/Chrome/26-03-2018/11-57-57/screenshots' exists but is a directory
	at org.apache.commons.io.FileUtils.doCopyFile(FileUtils.java:1118)
	at org.apache.commons.io.FileUtils.copyFile(FileUtils.java:1076)
	at org.apache.commons.io.FileUtils.copyFile(FileUtils.java:1028)
	at com.payswiff.util.DriverUtilsImpl.takescreenshot(DriverUtilsImpl.java:1547)
	... 25 more
]]>
    </error>
  </testcase> <!-- login -->
</testsuite> <!-- com.payswiff_Regression.test -->
